plugins/inventory/hrobot.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2019 Oleksandr Stepanov <alexandrst88@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+    name: hrobot
```

Comment:
I think it's kind of consistency of naming in the collection itself.

Reply:
The collection should have been called `robot`, but `robot` is too generic. We chose `hrobot` since `hetzner_robot` was a bit long. So in general, `robot` is a better name :)

Reply:
so if this plugin plugin will be rename as `robot` does it mean that all vars should be renamed as `robot_server_ip`?

Reply:
I think the prefix `hrobot_` is fine. You could also use `hetzner_robot_`, or just `hetzner_`, but I think `hrobot_` is also fine.

Reply:
so you think that plugin should be renamed to robot instead of hrobot?

Reply:
I think it would be better to call it `community.hrobot.robot`.